### ISSUE TITLE
Docs fixes before 1.4 release

### ISSUE THREAD

**Issue by MarcSkovMadsen** [holoviz/panel#6566]
https://holoviz-dev.github.io/panel/tutorials/basic/build_animation.html

![image](https://github.com/holoviz/panel/assets/42288570/e21398da-2f5f-488d-a29c-da5a509a2c34)

If the videos could be made working on iPhone too it would be great. Philipp mentioned it's probably are content-type issue. Whatever that means.

**Comment by MarcSkovMadsen** [comment]
There is a similar problem with the audio file here

https://holoviz-dev.github.io/panel/tutorials/basic/pn_panel.html

 ![image](https://github.com/holoviz/panel/assets/42288570/0e68f728-bf64-4f4a-9a0a-b45d19ef4f03)

**Comment by MarcSkovMadsen** [comment]
Maybe the comment below should be adjusted as we are not ready to recommend on.rx over pn.bind as discussed at HoloViz meeting https://holoviz-dev.github.io/panel/tutorials/basic/pn_rx.html

![image](https://github.com/holoviz/panel/assets/42288570/675d677e-64aa-4299-96e3-d62bce9a6205)

**Comment by MarcSkovMadsen** [comment]
I would recommend recommending studying the new Param references guide too. https://holoviz-dev.github.io/panel/tutorials/basic/pn_rx.html

![image](https://github.com/holoviz/panel/assets/42288570/a0d6da69-3f7e-4303-880f-23d072281f70)

**Comment by cdeil** [comment]
@MarcSkovMadsen - Maybe consider cross-linking the "monitoring dashboard" and "streaming dashboard" tutorials? Or put them next to each other?

I think that would make it easier for users to understand that those are two ways to achieve the same thing to have an auto-updating app, but the "monitoring" approach is simple (just add meta_refresh="2") but leads to full page refreshes and is not very efficient (I presume) both on the server and browser, whereas the "streaming" approach is more complicated to code but then more efficient and flexible (e.g. one could update different things at different frequencies, e.g. some every second, others only every minute or hour).

I would recommend to put the "Build a TODO app" at the end since it is quite complex (nine pn.bind calls) and could be a nice liftoff IMO then to the intermediate section to learn better coding patterns to build such an app.

Obviously just a matter of taste with the order, but I thought I'd share my thoughts after going through it.

<img width="582" alt="Screenshot 2024-03-25 at 11 11 22" src="https://github.com/holoviz/panel/assets/852409/7f03953e-2a46-4e12-94ef-c168b72d2899">

**Comment by philippjfr** [comment]
The mp4 and mp3 issues are persisting. I'm fairly certain it's some configuration that's missing in CloudFront or on the S3 buckets.

**Comment by MarcSkovMadsen** [comment]
> The mp4 and mp3 issues are persisting. I'm fairly certain it's some configuration that's missing in CloudFront or on the S3 buckets.

I believe the standalone links to the videos work on iPhone. But it's when used in video element in panel docs they don't work on iPhone.

**Comment by MarcSkovMadsen** [comment]
Yes my iPhone can play the link https://assets.holoviz.org/panel/tutorials/bar_chart_race.mp4.

![image](https://github.com/holoviz/panel/assets/42288570/13f66b6d-8219-4188-b8d6-ef30b65a3b76)

Just not when used in the video tag in the build_animation tutorial.

**Comment by MarcSkovMadsen** [comment]
This might be worth investigating.

To work with iOS the Accept-Ranges: bytes HTTP response header must be included, see: Safari Web Content Guide - Configuring Your Server. https://stackoverflow.com/questions/10511006/html5-video-not-working-on-ipad

**Comment by philippjfr** [comment]
Appears like we do return those headers:

> ` curl -I https://assets.holoviz.org/panel/tutorials/bar_chart_race.mp4`

```
HTTP/2 200
content-type: video/mp4
content-length: 181910
last-modified: Fri, 08 Mar 2024 05:36:00 GMT
x-amz-server-side-encryption: AES256
accept-ranges: bytes
server: AmazonS3
date: Mon, 25 Mar 2024 13:40:12 GMT
etag: "df845f2df17e17a937e05afa02ad7672"
vary: Accept-Encoding
x-cache: Hit from cloudfront
via: 1.1 cc4ec7b00e99f66f65c59b638212180e.cloudfront.net (CloudFront)
x-amz-cf-pop: TXL50-P1
x-amz-cf-id: 8bspr16bDOFnpDS3wXAmExDo_nlwDMk7yykPveL0vGBFN2qlMEB9hg==
age: 12686
vary: Origin
```
